packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -154,7 +154,6 @@ class IOSDevice extends Device {
     DebuggingOptions debuggingOptions,
     Map<String, dynamic> platformArgs,
     bool prebuiltApplication: false,
-    bool previewDart2: false,
```

Comment:
Also consider cleaning up 'previewDart2' argument in FuchsiaDevice.startApp() in fuchsia_device.dart.

Reply:
Done, thanks!